src/unix/linux_like/linux/gnu/b64/sparc64/mod.rs

```diff
@@ -973,6 +973,9 @@ pub const SYS_copy_file_range: ::c_long = 357;
 pub const SYS_preadv2: ::c_long = 358;
 pub const SYS_pwritev2: ::c_long = 359;
 pub const SYS_statx: ::c_long = 360;
+pub const SYS_pidfd_open: ::c_long = 434;
+// Reserved in the kernel, but not actually implemented yet
+pub const SYS_clone3: ::c_long = 435;
```

Comment:
Indeed, but this is fine since I marked `435` as reserved for all architectures that do not implement `clone3()` yet and so you're guaranteed that we don't take that number for another system call!

Reply:
We're in the process of updating our kernel docs where @cyphar and I will add a line like:
```diff
+In the past, syscall numbers were completely inconsistent
+between architectures, but a lot of work has gone into trying to unify the
+syscall tables. If you do not wish to enable your syscall on all architectures,
+add a comment to all other architectures' syscall tables to make it clear that
+the number has been reserved for your syscall.
```